mlflow/tracking/client.py

```diff
@@ -789,7 +790,7 @@ def start_span(
             trace_manager.register_span(span)
             return span
         except Exception as e:
-            _logger.warning(f"Failed to start span {name}: {e}")
+            _logger.warning(f"Failed to start span {name}: {get_stacktrace(e)}")
```

Comment:
does this print out the full traceback?

Reply:
Yes

Reply:
we can use `exc_info` instead.

Reply:
the full traceback is verbose and might scare users. can we use `_logger.debug("", exc_info=True)`?